assets/data.json

```diff
@@ -1,634 +1,689 @@
+  },
+  {
+    "raster": "hexagons/201920/mmh_demo_201920.png",
+    "name": "mmh_demo",
+    "description": "for the 2019/202 season season I demoed sticker",
+    "filename": "meta/201920/man_mat_hack_demo_201920.json"
+  },
+  {
+    "name": "MMH2",
+    "raster": "hexagons/201920/MMH2.png",
+    "description": "ManMetHacks2 Hex sticker for the 2019/202 season",
+    "filename": "meta/201920/MMH2.json"
+  },
+  {
+    "raster": "hexagons/201920/oxford-hacks.png",
+    "name": "oxford-hacks",
+    "description": "for the 2019/202 season season I demoed sticker",
+    "filename": "meta/201920/oxford-hacks.json"
+  },
+  {
+    "raster": "hexagons/201920/royal-hackway-201920.png",
     "name": "royal-hackway-201920",
-    "description": "2020 season I demoed sticker",
-    "filename": "meta/royal-hackway-201920.json"
+    "description": "for the 2019/202 season season I demoed sticker",
+    "filename": "meta/201920/royal-hackway-201920.json"
   },
   {
-    "raster": "hexagons/201617/ru_hacking.png",
-    "name": "ru_hacking",
-    "description": "2017 season I demoed sticker",
-    "filename": "meta/ru_hacking.json"
+    "raster": "hexagons/202021/AstonHack202021.png",
+    "name": "AstonHack202021",
+    "description": "2020/21 season I demoed sticker",
+    "filename": "meta/202021/AstonHack_202021.json"
   },
   {
-    "raster": "hexagons/201617/soton_hacks_2017.png",
-    "name": "soton_hacks_2017",
-    "description": "2017 season I demoed sticker",
-    "filename": "meta/soton_hacks_2017.json"
+    "raster": "hexagons/202021/Hack_the_Burgh_202021.png",
+    "name": "Hack_the_Burgh_202021",
+    "description": "2020/21 season I demoed sticker",
+    "filename": "meta/202021/Hack_the_Burgh_202021.json"
   },
   {
-    "name": "StacHackDemo16",
-    "raster": "hexagons/stac_hacks_demo_2016.png",
-    "description": "I demoed sticker for  Stac Hacks 2016",
-    "filename": "meta/stac_hacks_demo_2016.json"
+    "raster": "hexagons/202021/HackSurrey202021.png",
+    "name": "HackSurrey202021",
+    "description": "2020/21 season I demoed sticker",
+    "filename": "meta/202021/HackSurrey_202021.json"
   },
   {
-    "name": "StirHackDemo16",
-    "raster": "hexagons/stir_hack_demo_2016.png",
-    "description": "I demoed sticker for  Stir Hack 2016",
-    "filename": "meta/stir_hack_demo_2016.json"
+    "raster": "hexagons/202021/HexCambridge202021.png",
+    "name": "HexCambridge202021",
+    "description": "2020/21 season I demoed sticker",
+    "filename": "meta/202021/HexCambridge_202021.json"
   },
   {
-    "name": "StrathHackDemo16",
-    "raster": "hexagons/strath_hack_demo_2016.png",
-    "description": "I demoed sticker for  Strath Hack 2016",
-    "filename": "meta/strath_hack_demo_2016.json"
+    "raster": "hexagons/202021/RoyalHackaway202021.png",
+    "name": "RoyalHackaway202021",
+    "description": "2020/21 season I demoed sticker",
+    "filename": "meta/202021/RoyalHackaway_202021.json"
```

Comment:
These are all not showing up on https://ksouthern.github.io/hexbin/

Reply:
Should now be fixed (or at least they all show for me on https://ksouthern.github.io/hexbin/ now)